packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/dataservice/ProvenanceProxy.java

```diff
@@ -20,11 +21,12 @@ Response createProvenance(
 		Provenance provenance
 	);
 
-	@GET
-	@Path("/derived_from")
-	Response searchProvenance(
-		@QueryParam("artifact_id") String artifactId,
-		@QueryParam("artifact_type") String artifactType
+	@POST
+	@Path("/search")
+	@Consumes(MediaType.APPLICATION_JSON)
```

Comment:
I don't believe this Consumes is needed - that should be default type

Reply:
It doesn't hurt being a bit more specific but I will remove to simplify things

Reply:
done